src/app/clusters/level-control/level-control.cpp

```diff
@@ -478,11 +535,20 @@ static void writeRemainingTime(EndpointId endpoint, uint16_t remainingTimeMs)
         // This is done to ensure that the attribute, in tenths of a second, only
         // goes to zero when the remaining time in milliseconds is actually zero.
         uint16_t remainingTimeDs = static_cast<uint16_t>((remainingTimeMs + 99) / 100);
-        Status status            = LevelControl::Attributes::RemainingTime::Set(endpoint, remainingTimeDs);
-        if (status != Status::Success)
+        auto markDirty           = MarkAttributeDirty::kNo;
+        auto state               = getState(endpoint);
+        auto now                 = System::SystemClock().GetMonotonicTimestamp();
+
+        // Establish the quiet report condition for the RemainingTime Attribute
+        // The quiet report is determined by the previously set policies:
+        // - kMarkDirtyOnChangeToFromZero : When the value changes from 0 to any other value and vice versa, or
+        // - kMarkDirtyOnIncrement : When the value increases.
+        if (state->quietRemainingTime.SetValue(remainingTimeDs, now) == AttributeDirtyState::kMustReport)
         {
-            ChipLogProgress(Zcl, "ERR: writing remaining time %x", to_underlying(status));
+            markDirty = MarkAttributeDirty::kIfChanged;
         }
+
+        Attributes::RemainingTime::Set(endpoint, state->quietRemainingTime.value().ValueOr(0), markDirty);
```

Comment:
Why the `ValueOr(0)`?  We just set it to a non-null value, so why is this not just `Value()`?